lib/uo_cli.rb

```diff
@@ -89,19 +102,27 @@ module CLI
       puts e
     end
 
-
-    # Simulate energy usage for each Feature in the Scenario\
+    # Simulate energy usage for each Feature or for single feature as defined by ScenarioCSV\
     # params\
     # +scenario+:: _string_ Path to csv file that defines the scenario\
     # +feature_file_path+:: _string_ Path to Feature File used to describe set of features in the district
     # 
     # FIXME: This only works when scenario_file and feature_file are in the project root directory
     # Also, feels a little weird that now I'm only using instance variables and not passing anything to this function. I guess it's ok?
-    def self.run_func
-        name = "#{@scenario_name.split('.')[0].capitalize}"
-        root_dir = File.absolute_path(@scenario_path)
+    def self.run_func 
+        name = "#{@scenario_folder}"
+        root_dir = File.dirname(File.absolute_path(@user_input[:scenario]))
         run_dir = File.join(root_dir, 'run', name.downcase)
-        csv_file = File.join(root_dir, @scenario_name)
+
+        if @feature_id
```

Comment:
I'm getting an error here when re-running a single datapoint that has already been run as a part of the scenario. `...uo_cli.rb:121:in `run_func': uninitialized constant URBANopt::CLI::FileUtiles (NameError)`

Reply:
@vtnate Try now. It was a type, it should be FileUtils on line 112